install/kubernetes/helm/istio/charts/gateways/templates/deployment.yaml

```diff
@@ -234,6 +234,9 @@ spec:
           {{- if $spec.sds.enabled }}
           - name: ISTIO_META_USER_SDS
             value: "true"
+          - name: ISTIO_METAJSON_STATS
+            value: |
+              {"sidecar.istio.io/statsInclusionSuffixes": "ssl_context_update_by_sds"}
```

Comment:
What happens if the user specifies `ISTIO_METAJSON_STATS` themselves and they set sds.enabled? Now `ISTIO_METAJSON_STATS` is duplicated, which I would guess causes the install to fail

Reply:
If user specifies ISTIO_METAJSON_STATS and enables sds.enabled, is the sds annotation appended to the annotation specified by user? I am not clear how it works. @mandarjog could you share some insights?

Reply:
@JimmyCYJ do you think that this should part of the `default` inclusion set of the proxy?

If so please do it thru code, not config.  https://github.com/istio/istio/blob/master/pkg/bootstrap/bootstrap_config.go#L77
If this is a single stat, it may not even have a whole lot of performance hit.

The annotation is meant for customer use, not internal use.